### PR TITLE
Adding aro for 4.15

### PR DESCRIPTION
**Pull request by paigerube14**

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416 periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-aro-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415

**Comment by paigerube14** [comment]
@shahsahil264 @chaitanyaenr PTAL, the aro tests are passing for 4.15 

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/50795/rehearse-50795-periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415/1807855843354349568

File: ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.15-nightly.yaml [hunk 1]
```diff
@@ -95,6 +99,29 @@ tests:
...
+      ARO_MASTER_VM_SIZE: Standard_D16s_v3
+      ARO_WORKER_VM_SIZE: Standard_D8s_v3
+      CHURN: "false"
+      GC: "false"
+      ITERATION_MULTIPLIER_ENV: "4"
+      PROFILE_TYPE: reporting
+      TELEMETRY_GROUP: prow-ocp-aro-4.15
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    - chain: redhat-chaos-krkn-hub-etcd-tests
```
Comment (chaitanyaenr) [inline ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.15-nightly.yaml:117]: We can replace the test cases with krkn-hub chain: https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-chaos/krkn-hub-tests/redhat-chaos-krkn-hub-tests-chain.yaml + hog-scenarios. Thoughts?

**Review by shahsahil264 — APPROVED** [review]
LGTM

File: ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.15-nightly.yaml (hunk 1 above)
Comment (paigerube14) [reply ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.15-nightly.yaml:117]: the prometheus chain doesnt work for aro: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/50795/rehearse-50795-periodic-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-aro-415/1818687176120995840

**Pull request merged**